src/__tests__/forms.js

```diff
@@ -32,29 +34,24 @@ function Login({onSubmit}) {
 test('login form submits', () => {
   const fakeUser = {username: 'jackiechan', password: 'hiya! 🥋'}
   const handleSubmit = jest.fn()
-  const {container, getByLabelText, getByText} = render(
-    <Login onSubmit={handleSubmit} />,
-  )
+  const {getByLabelText, getByText} = render(<Login onSubmit={handleSubmit} />)
 
   const usernameNode = getByLabelText(/username/i)
   const passwordNode = getByLabelText(/password/i)
-  const formNode = container.querySelector('form')
   const submitButtonNode = getByText(/submit/i)
 
   // Act
   usernameNode.value = fakeUser.username
   passwordNode.value = fakeUser.password
   // NOTE: in jsdom, it's not possible to trigger a form submission
   // by clicking on the submit button. This is really unfortunate.
-  // So the next best thing is to simulate a submit on the form itself
+  // So the next best thing is to fire a submit on the form itself
   // then ensure that there's a submit button.
-  Simulate.submit(formNode)
+  fireEvent.click(submitButtonNode)
```

Comment:
According to the `NOTE` comment above, clicking on the submit button won't work in `jsdom`. Does `fireEvent.click` take care of that?